modules/ROOT/pages/release-notes.adoc

```diff
@@ -304,12 +299,21 @@ With the new Self-Contained Application (abbr. SCA) approach, creating or updati
 
 == Configuration changes
 
+=== Update only startup
+
+A new `bonita.runtime.startup.update-only` configuration boolean property has been added to perform an update only startup.
+When enabled, the application will start, go through its initialization and update phase and exit afterward. Exit status can be successful (`0`) or in error (`1`).
+
 === Runtime property renaming
 
 In order to improve Bonita property naming coherence, a work is in progress to change some property names.
 In this release, the following properties have been renamed:
 
 * [.line-through]#`bonita.tenant.session.duration`# has been renamed to `bonita.runtime.session.duration`. If you happened to customize this property, please update it in file `bonita-tenant-community-custom.properties` (The old property is **still supported** but will be removed in a later version)
+* [.line-through]#`userName`# has been renamed to `bonita.runtime.admin.username`. If you happened to customize this property, please update it in file `bonita-tenant-community-custom.properties`
+* [.line-through]#`userPassword`# has been renamed to `bonita.runtime.admin.password`. If you happened to customize this property, please update it in file `bonita-tenant-community-custom.properties`
```

Comment:
@rbioteau This is already handled by the Update Tool.
https://github.com/bonitasoft/bonita-migration-sp/pull/424